karrot/groups/templates/user_got_role.subject.jinja2

```diff
@@ -0,0 +1 @@
+{% trans group_name=group_name|safe %}You received a new role in {{ group_name }}!{% endtrans %}
```

Comment:
Comment only: most email subjects start with `[{{ site_name }}]` - although not the other trust ones.

I think `[{{ group_name }}]` would actually be better, so the identity is more focused on the group, not on the site, might be possible to do it at a higher level, to always add that to the subject, where there is a group set ...